web/src/pages/TodayPage/TodayPage.tsx

```diff
@@ -60,7 +60,7 @@ const TodayPage = () => {
                 <Checkbox className="border-primary-foreground bg-black/20 hover:bg-secondary" />
                 <div className="absolute left-10 top-2.5 flex gap-1">
                   <span>6</span>
```

Comment:
I just noticed that the value here is hard-coded---opening an issue to fix this!